volumes/short_term_counting_program/move_match/speedvol_stats.sql

```diff
@@ -0,0 +1,251 @@
+-- This code replicates statistics from the speed volume Speed Percentile Report: https://move.intra.prod-toronto.ca/view/location/s1:Ab_DAA/POINTS/reports/ATR_SPEED_VOLUME
+-- Note that am and pm peak hours as presented in this report are based on clock face hours (not 15 minute bins)
```

Comment:
This should be changed to be consistent across count types (using a rolling hour window instead of hourly aggregation)